src/org.freedesktop.IBus.service.in

```diff
@@ -0,0 +1,28 @@
+[Unit]
+Description=IBus Daemon
+CollectMode=inactive-or-failed
+
+# Require GNOME session and specify startup ordering
+Requisite=gnome-session-initialized.target
+After=gnome-session-initialized.target
+PartOf=gnome-session-initialized.target
+Before=gnome-session.target
+
+# Needs to run when DISPLAY/WAYLAND_DISPLAY is set
+After=gnome-session-initialized.target
+PartOf=gnome-session-initialized.target
+
+# Never run in GDM
+Conflicts=gnome-session@gnome-login.target
+
+[Service]
+Type=dbus
+# Only pull --xim in X11 session, it is done via Xwayland-session.d on Wayland
+ExecStart=sh -c '@bindir@/ibus-daemon --panel disable $([[ $XDG_SESSION_TYPE == "x11" ]] && echo "--xim")'
+Restart=on-abnormal
+BusName=org.freedesktop.IBus
```

Comment:
If we prepare the service file by desktop, probably I think `org.freedesktop.IBus.session.GNOME.service` file would be better for this service name than `org.freedesktop.IBus.service` file?

As you may notice https://github.com/ibus/ibus/pull/1896/files, probably I wish to add  `org.freedesktop.IBus.service` file which includes $IBUS_ARGS options for ibus-dameon for enabled panel & verbose options.

Reply:
Renamed the systemd service to `org.freedesktop.IBus.session.GNOME.service`, and updated all references to the filename.